packages/wxt/src/core/builders/vite/plugins/devHtmlPrerender.ts

```diff
@@ -113,29 +113,30 @@ export function devHtmlPrerender(
     {
       name: 'wxt:virtualize-inline-scripts',
       apply: 'serve',
-      resolveId(id) {
-        // Resolve inline scripts
-        if (id.startsWith(virtualInlineScript)) {
-          return '\0' + id;
-        }
-
-        // Ignore chunks during HTML file pre-rendering
-        if (id.startsWith('/chunks/')) {
-          return '\0noop';
-        }
+      resolveId: {
+        filter: {
+          id: [new RegExp(`^${virtualInlineScript}`), /^\/chunks\//],
+        },
+        handler(id) {
+          // Resolve inline scripts
+          if (id.startsWith(virtualInlineScript)) {
+            return '\0' + id;
+          } else {
+            return '\0noop';
+          }
+        },
       },
-      load(id) {
-        // Resolve virtualized inline scripts
-        if (id.startsWith(resolvedVirtualInlineScript)) {
-          // id="virtual:wxt-inline-script?<key>"
+      load: {
+        filter: {
+          id: {
+            include: [new RegExp(`^${resolvedVirtualInlineScript}`)],
+            exclude: '\0noop',
```

Comment:
Your include set has no union with exclude set, so it's safe to remove this exclude filter

Reply:
Oh yeah, that's true, i'll remove it. Thanks